src/main/java/frc/robot/util/PathPlannerCommands.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.util;
+
+import com.pathplanner.lib.commands.FollowPathHolonomic;
+import com.pathplanner.lib.path.PathPlannerPath;
+import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
+import com.pathplanner.lib.util.ReplanningConfig;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystem.DriveSubsystem;
+import frc.robot.subsystem.PositioningSubsystem;
+
+/** Add your docs here. */
+public class PathPlannerCommands {
+    private static final ReplanningConfig REPLANNING_CONFIG = new ReplanningConfig();
+
+    public static Command getFollowPathCommand(
+            DriveSubsystem drive,
+            PositioningSubsystem positioning,
+            PathPlannerPath path,
+            boolean shouldAssumeRobotIsAtStart) {
+        if (DriverStation.getAlliance().map(a -> a == Alliance.Red).orElse(false)) {
+            path = path.flipPath();
+        }
+
+        final PathPlannerPath thePath = path;
```

Comment:
Minor nitpick, but I'd add a comment explaining why this is necessary.

I assume it's because you can't bind the mutable `path` variable in the lambda used to reset the robot pose. But this is not immediately obvious.

It might also be nice to change this from a `PathPlannerPath` to a `Pose2d`. That is, instead of storing the whole path, store just the initial position (since that's all the lambda cares about anyway). Right now we have two variables in the same scope that refer to the same object (`path` and `thePath`). It would be nicer to just keep `path`, and change this to be something like `initialPose`.